lib/PTEVal.ml

```diff
@@ -108,6 +127,13 @@ let compare =
     lex_compare (fun p1 p2 -> Misc.int_compare p1.af p2.af) cmp in
   let cmp =
     lex_compare (fun p1 p2 -> String.compare p1.oa p2.oa) cmp in
+  let cmp =
+    lex_compare (fun p1 p2 -> List.fold_right2 (fun s1 s2 a ->
+                                  match a with
+                                  | 0 -> String.compare s1 s2
+                                  | _ -> a)
+                                p1.attrs p2.attrs 0)
```

Comment:
This comparison function for lists of strings appears elsewhere at least once (`symbConstant.ml`). One may think about moving it to the `Misc` module and sharing code.

Reply:
Ack, the new version has resolved this.